lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -84,6 +85,16 @@ private static IConfiguration GetRuntimeConfiguration(string name)
                     return YARNClientConfiguration.ConfigurationModule.Build();
                 case YARNRest:
                     return YARNClientConfiguration.ConfigurationModuleYARNRest.Build();
+                case HDInsight:
```

Comment:
@anupam128 Actually, I just saw this. Should we remove this? Is it possible for you to modify the HelloREEF example such that it takes in these arguments?